onnxruntime/test/onnx/TestCase.cc

```diff
@@ -177,6 +177,7 @@ using PATH_STRING_TYPE = std::basic_string<PATH_CHAR_TYPE>;
 class OnnxModelInfo : public TestModelInfo {
```

Comment:
BTW, the interface TestModelInfo  has two implementations: one for onnx, another for Tensorflow.

Reply:
Appreciate the info!